.editorconfig

```
@@ -0,0 +1,536 @@
+[*]
+charset = utf-8
+end_of_line = crlf
```

Shouldnt this be just linefeed as our deployments would be on a linux box. Eg. an executable script will most likely error out with crlf 🤔
IDE's are anyways okay with both formats.

```suggestion
end_of_line = lf
```